CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ MESSAGE(STATUS "using build environment '${NOF_BUILD_ENVIRONMENT}' with dependen
 
 project(nearobject-framework 
   LANGUAGES CXX
-  VERSION 0.3.0
+  VERSION 0.4.0
```

Comment:
In future, there should be a separate commit for the version bump. Normally, we'd bump it in the release branch, then have merge it back to `develop`. I didn't document this in the release process, though, so next time :)

Reply:
I ended up cherry-picking just that commit into develop, so it's all good now.